.gitignore

```
@@ -19,7 +19,7 @@ docker/srcs/kind/kind-config.yaml
 docker/srcs/uwsgi-django/sphinx/build
 docker/srcs/uwsgi-django/django_debug.log
 # 再度修正。entrypoint.shで最新ファイルの自動収集を設定したため。
-static/
+/static/
```

```suggestion
docker/srcs/uwsgi-django/static/
```

```
docker/srcs/uwsgi-django/static
```
のignoreなので、この記述で良いかもしれません。
念のためご確認お願いします！

---

勘違いしてました。その通り、pjnorootです
docker/srcs/uwsgi-django/static